attribute_standardizer/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.1"
+__version__ = "0.1.0"
```

Comment:
I had released it on [PyPI](https://pypi.org/project/attribute-standardizer/#history) in both these versions (0.1.1 and 0.1.0). 
Later, based on what @nsheff suggested, I didn't release the further updates on PyPI.
So, for this new release, technically the version should be upgraded right @khoroshevskyi ? Especially if we're going to make this one available on PyPI now

Reply:
I think we want to rename it to bedms, to be consistent with repository name. So it will have new version, and this will be the first official release